source/form.html

```diff
@@ -0,0 +1,140 @@
+            <label class="form__label" for="achievements-selfie">
+              <input class="form__checkbox" type="checkbox" name="achievements-selfie" id="achievements-selfie" checked> Сделал селфи с акулой
+            </label>
+            <label class="form__label" for="achievements-beach">
+              <input class="form__checkbox" type="checkbox" name="achievements-beach" id="achievements-beach" checked> Обгорел на пляже
+            </label>
+            <label class="form__label" for="achievements-chuck-norris">
+              <input class="form__checkbox" type="checkbox" name="achievements-chuck-norris" id="achievements-chuck-norris"> Видел Чака Норриса
+            </label>
+            <label class="form__label" for="achievements-souvenirs">
+              <input class="form__checkbox" type="checkbox" name="achievements-souvenirs" id="achievements-souvenirs"> Накупил сувениров
+            </label>
+            <label class="form__label" for="achievements-tower">
+              <input class="form__checkbox" type="checkbox" name="achievements-tower" id="achievements-tower" checked> Удержал башню
+            </label>
+            <label class="form__label" for="achievements-hotel">
+              <input type="checkbox" name="achievements-hotel" id="achievements-hotel"> Разгромил отель
+            </label>
+          </fieldset>
+          <fieldset class="form__contact">
+            <legend class="form__group-title">Контактная информация</legend>
+            <label class="form__label" for="phone">Номер телефона</label>
+            <input class="form__input" type="tel" name="phone" id="phone" placeholder="Номер, пожалуйста">
+            <label class="form__label" for="email">Адрес почты</label>
+            <input class="form__input" type="email" name="email" id="email" placeholder="Введите почту *" required>
+          </fieldset>
+          <fieldset class="form__travel">
+            <legend class="form__group-title">С каким приложением путешествовали?</legend>
+            <label class="form__label" for="travel-app-ios">
+              <input class="form__radio" type="radio" name="travel-app" id="travel-app-ios" checked> Pink для iOS
+            </label>
+            <label class="form__label" for="travel-app-android">
+              <input class="form__radio" type="radio" name="travel-app" id="travel-app-android"> Pink на Android
+            </label>
+            <label class="form__label" for="travel-app-windows">
+              <input class="form__radio" type="radio" name="travel-app" id="travel-app-windows"> Windows Phone
+            </label>
+          </fieldset>
+          <fieldset class="form__comment comment">
+            <label class="comment__title" for="comment-area">Опишите свои эмоции</label>
+            <textarea class="comment__area" name="comment-area" id="comment-area" rows="7" placeholder="Можно прям в красках, не стесняясь в выражениях"></textarea>
```

Comment:
- Лучше задавать ширину/высоту `textarea` через  **width/height** из css. **Сols и rows** лучше опустить, это редко используется и хуже читается. 
```suggestion
            <textarea class="comment__area" name="comment" id="comment-area" rows="7" placeholder="Можно прям в красках, не стесняясь в выражениях"></textarea>
```